### ISSUE TITLE
Les dossiers moodle vides ne peuvent pas être compressés en zip?

### ISSUE THREAD

**Issue by rakruk** [sebastienlabine/clone-moodle#12]
```
  File "D:/gitClones/clone-moodle/clone-moodle.py", line 207, in <module>
    main()
  File "D:/gitClones/clone-moodle/clone-moodle.py", line 195, in main
    findAllDocuments(resourceHTML, loginCookie)
  File "D:/gitClones/clone-moodle/clone-moodle.py", line 144, in findAllDocuments
    zip = zipfile.ZipFile(folderNameCleaned + ".zip")
  File "C:\Users\---\AppData\Local\Programs\Python\Python36\lib\zipfile.py", line 1113, in __init__
    self.fp = io.open(file, filemode)
FileNotFoundError: [Errno 2] No such file or directory: 'Chapitre 1 électrostatique.zip'
```
Effectivement, le fichier `'Chapitre 1 électrostatique.zip'` n’apparaît pas sur mon disque. Au départ j'ai pensé que c'était car il fut enregistré sous un autre nom, mais aucune trace d'un fichier zip dont le nom ressemble, pourtant le dossier a été créé sous le même nom. En regardant ce qui a été téléchargé on comprend mieux, le dossier est vide, tout comme le dossier dans le moodle, **ce qui me laisse croire que zipfile essaie, et échoue à compresser un dossier vide et donc ne fait rien.** Ce qui explique pourquoi il est impossible de l'ouvrir.

**Comment by sebastienlabine** [comment]
Effectivement, on dirait que les dossiers vides ne peuvent pas être ajouté... Voici une issue sur stackoverflow qui référence ce problème. Je vais essayer après mes intras de fix ce bug!

https://stackoverflow.com/questions/11751822/how-do-i-create-a-zip-file-of-a-file-path-using-python-including-empty-director/11751948#11751948
